o2.sh

```diff
@@ -62,6 +62,7 @@ valid_defaults:
   - o2-dev-fairroot
   - alo
   - o2-prod
+  - o2-ninja
```

Comment:
Sorry for being late. How about overloading this on the dev-fairroot one? Having defaults to test different tools diverges rather quickly, IMHO.

Reply:
That's precisely what has been discussed if you read the conversation. However, at the moment dev-fairroot does not work for other reasons. I'd rather have it fixed and _then_ I agree to have it on dev-fairroot, that's the plan.